builder/dockerfile/parser/testfiles/health/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM debian
+ADD check.sh main.sh /app/
+CMD /app/main.sh
+HEALTHCHECK
+HEALTHCHECK --interval=5s --timeout=3s --retries=1 \
```

Comment:
nit: I think a good practice is multiple failures like 3 to tolerate random failures. Giving an example of `--retries=1` might not be appropriate.